starboard/extension/media_settings.h

```diff
@@ -36,6 +36,10 @@ typedef struct StarboardExtensionMediaSettingsApi {
   // TV.
   void (*EnableAsyncReleaseMediaCodecBridge)(bool value);
 
+  // This API set the timeout seconds for async release MediaCodecBridge on
```

Comment:
updated. 

"Android TV" is not removed because the name "StarboardExtensionMediaSettingsApi" and "SetAsyncReleaseMediaCodecBridgeTimeoutSeconds" do not indicate it is ATV, I 'd give people context through the comment.